stdlib/public/Concurrency/Actor.cpp

```diff
@@ -1617,9 +1617,9 @@ static void swift_task_switchImpl(SWIFT_ASYNC_CONTEXT AsyncContext *resumeContex
   // executor.
   SWIFT_TASK_DEBUG_LOG("switch failed, task %p enqueued on executor %p", task,
                        newExecutor.getIdentity());
-  task->flagAsSuspended();
+
+  task->flagAsEnqueuedOnExecutor(newExecutor);
```

Comment:
From the names, I'd expect these methods to just be bookkeeping.  If they're doing the actual work of enqueuing the task, they should probably have a verb as their name, probably `enqueue`.

Reply:
Yea I was conflicted as well - it felt really nice to keep it in line with the other `flagAs` functions since they nicely encapsulate all the state changes of a task. I can make this clean-up in a follow on PR.

Reply:
Okay.  Originally these methods were all just bookkeeping, so since they've now all become the actual way to do things, they should all be renamed.

Reply:
I will do that in my next PR.